pyrefinebio/compendia.py

```diff
@@ -0,0 +1,81 @@
+import pyrefinebio.computed_file as prb_computed_file
+from pyrefinebio.http import get_by_endpoint
+from pyrefinebio.util import generator_from_pagination
+
+
+class Compendia:
+    """Compendia 
+
+    Retrieve a compendium result based on id
+
+        ex:
+        >>> import pyrefinebio
+        >>> id = 1
+        >>> sample = pyrefinebio.Compendia.get(id)
+
+    Retrieve a list of compendium results based on filters
+
+        ex:
+        >>> import pyrefinebio
+        >>> samples = pyrefinebio.Compendia.search(compendium_version="")
+    """
+
+    def __init__(
+        self,
+        id=None,
+        primary_organism_name=None,
+        organism_names=None,
+        svd_algorithm=None,
+        quant_sf_only=None,
+        compendium_version=None,
+        computed_file=None,
+    ):
+        self.id = id
+        self.primary_organism_name = primary_organism_name
+        self.organism_names = organism_names
+        self.svd_algorithm = svd_algorithm
+        self.quant_sf_only = quant_sf_only
+        self.compendium_version = compendium_version
+        self.computed_file = prb_computed_file.ComputedFile(**(computed_file or {}))
```

Comment:
I think it'd be better to have `self.computed_file` be None if it doesn't exist rather than an empty object. (Cause then things like `if self.computed_file:` wouldn't work right.)
```suggestion
        self.computed_file = prb_computed_file.ComputedFile(**computed_file) if computed_file else None
```